lib/BackgroundJob/Tasks/ImageProcessingTask.php

```diff
@@ -151,6 +151,8 @@ public function execute(FaceRecognitionContext $context) {
 		$dataDir = rtrim($context->config->getSystemValue('datadirectory', \OC::$SERVERROOT.'/data'), '/');
 		$images = $context->propertyBag['images'];
 
+		$this->logInfo('File ' . $requirements->getFaceDetectionModel() . ': '. filesize($requirements->getFaceDetectionModel())/(1024*1024) . ' Mb');
```

Comment:
Try to print size to compare it and make sure that file exists ..
Here's where it fails.. or if I delete this line, it fails when try to open the model.